package.json

```diff
@@ -10,23 +10,29 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@bundlr-network/client": "^0.7.1",
     "@heroicons/react": "^1.0.5",
     "@metaplex/js": "^4.12.0",
     "@solana/buffer-layout": "^4.0.0",
     "@solana/spl-token": "^0.2.0",
     "@solana/wallet-adapter-base": "^0.9.3",
+    "@solana/wallet-adapter-phantom": "^0.9.3",
```

Comment:
You're probably right. I noticed the Bundlr demo directly imported it, so that's what I did to just make sure it worked. I can try to refactor to avoid this double import.